src/Form/Ingest/Review.php

```diff
@@ -145,13 +145,19 @@ public function submitActivation(array &$form, FormStateInterface $form_state) {
    *   A reference to the batch context.
    */
   public function runBatchOp(MigrationInterface $migration, MigrateExecutable $e, &$context) {
+    assert($e instanceof MigrateBatchExecutable);
     $sandbox =& $context['sandbox'];
 
     if (!isset($sandbox['prepped'])) {
       $context['results']['status'] = NULL;
 
       try {
         $e->prepareBatch();
+        if (method_exists($e, 'getQueueSize')) {
+          // Method was introduced in dgi_migrate:3.16, the later use of
```

Comment:
Should be updated according to whatever version https://github.com/discoverygarden/dgi_migrate/pull/162 creates.